### PR TITLE
Feat: support outputs for addon

### PR DESCRIPTION
**Pull request by wangyikewxgm**
Signed-off-by: 楚岳 <wangyike.wyk@alibaba-inc.com>

### Description of your changes

Fixes #

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: pkg/addon/render.go [hunk 1]
```diff
@@ -275,6 +277,28 @@ func checkNeedAttachTopologyPolicy(app *v1beta1.Application, addon *InstallPacka
 	return true
 }
 
+func renderAliasOutputs(addon *InstallPackage, args map[string]interface{}) ([]*unstructured.Unstructured, error) {
```
Comment (wonderflow) [inline pkg/addon/render.go:280]:
```suggestion
func renderOutputs(addon *InstallPackage, args map[string]interface{}) ([]*unstructured.Unstructured, error) {
```

File: pkg/addon/addon.go [hunk 2]
```diff
@@ -1084,6 +1084,12 @@ func (h *Installer) dispatchAddonResource(addon *InstallPackage) error {
 		return errors.Wrapf(err, "cannot pass definition to addon app's annotation")
 	}
 
+	var aliasOutputs []*unstructured.Unstructured
+	aliasOutputs, err = renderAliasOutputs(addon, h.args)
```
Comment (wonderflow) [inline pkg/addon/addon.go:1088]: aliasOutputs => AuxiliaryOutputs

File: pkg/addon/render.go [hunk 3]
```diff
@@ -275,6 +277,28 @@ func checkNeedAttachTopologyPolicy(app *v1beta1.Application, addon *InstallPacka
...
+	outputs := map[string]interface{}{}
+	r := addonCueTemplateRender{
+		addon:     addon,
+		inputArgs: args,
+	}
+	if err := r.toObject(addon.AppCueTemplate.Data, renderAliasOutputsPath, &outputs); err != nil {
+		if err.Error() == "var(path=outputs) not exist" {
```
Comment (wonderflow) [inline pkg/addon/render.go:287]: it should be a function as widely used

File: pkg/addon/render.go [hunk 4]
```diff
@@ -275,6 +277,28 @@ func checkNeedAttachTopologyPolicy(app *v1beta1.Application, addon *InstallPacka
...
+		addon:     addon,
+		inputArgs: args,
+	}
+	if err := r.toObject(addon.AppCueTemplate.Data, renderAuxiliaryOutputsPath, &outputs); err != nil {
+		if isErrorCueRenderPathNotFound(err, renderAuxiliaryOutputsPath) {
+			return nil, nil
+		}
+		return nil, err
+	}
+	fmt.Println(outputs)
```
Comment (wonderflow) [inline pkg/addon/render.go:292]: debug

Comment (wangyikewxgm) [reply pkg/addon/render.go]: fixed

**Review by wonderflow — APPROVED** [review]

**Pull request merged**